.github/pull_request_template.md

```diff
@@ -1,18 +1,19 @@
-Summary:
+**Summary**:
 
-Related issue: 
+**Related issue**: 
 
-Suggested reviewer(s):
+**Suggested reviewer(s)**:
 
-Checklist:
+**Checklist**:
 
 - [ ] The headers of all files contain a reference to the repository license (i.e., "License: This file is part of py-mmd-tools, licensed under the Apache License 2.0 (https://www.apache.org/licenses/LICENSE-2.0)")
 - [ ] The test coverage increased or remained the same as before
```

Comment:
Even though the S-ENDA definition of done asks for 100% test coverage?

Reply:
In practice we're often below 100% - but this would anyway be valid, wouldn't it? I mean - we wouldn't accept going from 100% to 99%...